### ISSUE TITLE
[lodash] Remove usage of _.forIn

### ISSUE THREAD

**Issue by abdavid** [moleculerjs/moleculer#746]
See #433 for context.

**Describe the solution you'd like**
Remove usage of `_.forIn` throughout Moleculer. 

**Describe alternatives you've considered**
Alternatives:
```
// 1
for (variable of iterable) {
  statement
}

// 2
for (variable in object)
  statement

// 3
for await (variable of asyncIterable) {
  statement
}
```

... and a myriad of other natively supported patterns which could be used depending on the context. 

**Additional context**
https://github.com/moleculerjs/moleculer/search?q=_.forIn&unscoped_q=_.forIn

**Comment by icebob** [comment]
What is the difference between the first two alternatives?

_By the way, we can't use the third because we don't use async/await in the core modules because await is slower then Promise, and performance is important in Moleculer :)_

**Comment by abdavid** [comment]
> What is the difference between the first two alternatives?
> 
> _By the way, we can't use the third because we don't use async/await in the core modules because await is slower then Promise, and performance is important in Moleculer :)_

**Alternative 1 is for objects that implement the iterable interface. Arrays, strings etc**

```javascript
> for(const a of "abcdefg")
... console.log(a)
a
b
c
d
e
f
g

```

```javascript
> for(const a of [1,2,3,4,5])
... console.log(a)
1
2
3
4
5

```

```javascript
> for(const a of {a:1,b:2,c:3})
... console.log(a)
Thrown:
TypeError: {(intermediate value)(intermediate value)(intermediate value)} is not iterable
```


**Alternative two is for "POJOs"**

```javascript
> for(const a in {a:1,b:2,c:3})
... console.log(a)
a
b
c
```

```javascript
// for in with Iterables "a" is index.
> for(const a in [1,2,3,4,5])
... console.log(a)
0
1
2
3
4
```

**Comment by abdavid** [comment]
Some considerations aswell:

https://jsperf.com/for-of-vs-for-loop
https://jsperf.com/for-vs-for-in

**Comment by icebob** [comment]
Yeah, the native `for(;;)` is the quickest, but we need to use better performance solution in the critical parts (action calling, event sending, transfer packets....etc). 

So if there is a loop which processes something from the broker options only once at the startup, we can use any solution because it is not cut down the runtime performance.

**Comment by Wallacy** [comment]
BTW: https://v8.dev/blog/fast-async
